.github/workflows/releases.yaml

```diff
@@ -0,0 +1,57 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - run: npm install
+      - run: npm build
+      - name: Store build files
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist-files
+          path: |
+            dist
+            src
+            README.md
+            package.json
+            LICENSE
+
+  release:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Download build files
+        uses: actions/download-artifact@v2
+        with:
+          name: dist-files
+      - name: Release
+        uses: softprops/action-gh-release@v1
```

Comment:
why bother with gh releases for a npm package?

Reply:
It appears to be best practice, as amqp.node, express, dotenv, sequelize etc does it